src/inputFileReader/inputFileReader.cc

```diff
@@ -729,6 +729,8 @@ inputFileReader::declare_parameters(dealii::ParameterHandler    &parameter_handl
     dealii::Patterns::Anything(),
     "The filename (not including the '.vtk' extension) for the file holding "
     "the grain structure to be loaded.");
+    
+  parameter_handler.declare_entry("vtk file type","UNSTRUCTURED",dealii::Patterns::Anything(),"Whether to load a unstructured file for grain structure.");
 
```

Comment:
I think a multiple selection pattern would be better here.
https://www.dealii.org/current/doxygen/deal.II/classPatterns_1_1MultipleSelection.html